src/app/elections/route.ts

```diff
@@ -1,27 +0,0 @@
-import { PrismaClient } from "@prisma/client";
```

Comment:
dont we need this file route anymore?

Reply:
We're trying to practice SSR and server functions in this project Marco, we're migrating things that were previously routes to this new logic.